scalr/manifest.json

```diff
@@ -0,0 +1,52 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "d74ce5c8-4e5a-485a-be79-ff55f8205c9d",
+  "app_id": "scalr",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Scalr is a Terraform Automation and COllaboration (TACO) product",
+    "title": "Scalr",
+    "media": [],
```

Comment:
We highly recommend including images like dashboard images, graphs, etc. Could you please add images?

Reply:
As mentioned on the call, this would be images of the current dashboard once you make the needed changes to the dashboard :)